app/services/foreman/access_permissions.rb

```diff
@@ -351,7 +351,7 @@
                                                :externalNodes, :pxe_config, :storeconfig_klasses, :auto_complete_search, :bmc,
                                                :runtime, :resources, :templates, :overview, :nics],
                                     :dashboard => [:OutOfSync, :errors, :active],
-                                    :unattended => [:host_template, :hostgroup_template],
+                                    :unattended => [:host_template, :hostgroup_template] + TemplateKind.default_template_labels.keys.collect(&:to_sym),
```

Comment:
This shouldn't be necessary now, since the action is always going to be either :host_template or :hostgroup_template - the routing changed in b1997f5 to route the template kinds to a single action.